solidity/contracts/PhasedEscrow.sol

```diff
@@ -71,3 +71,82 @@ contract CurveRewardsEscrowBeneficiary is Ownable {
         curveRewards.notifyRewardAmount(amount);
     }
 }
+
+/// @dev Interface of recipient contract for approveAndCall pattern.
+interface IStakerRewards { function receiveApproval(address _from, uint256 _value, address _token, bytes calldata _extraData) external; }
+
+/// @title BeaconBackportRewardsEscrowBeneficiary
+/// @notice A beneficiary contract that can receive a withdrawal phase from a
+///         PhasedEscrow contract. Immediately stakes the received tokens on a
+///         designated BeaconBackportRewardsEscrowBeneficiary contract.
+contract BeaconBackportRewardsEscrowBeneficiary is Ownable {
```

Comment:
I don't think beneficiaries have to be `Ownable`. Same for `CurveRewardsEscrowBeneficiary` - I think we missed it. Is there any reason for having them `Ownable`?

Reply:
Unless we are missing `onlyOwner` modifier in `__escrowSentTokens`?

Reply:
Yeah, I took another look and we could go without `Ownable` for staker rewards but it's a good move to go with `Ownable` and `onlyOwner` on `__escrowSentTokens` for `CurveRewardsEscrowBeneficiary` as their `notifyRewardAmount` has `onlyRewardDistribution` modifier.

Let's do it consistend for all beneficiaries and add `onlyOwner` to `__escrowSentTokens` and make `PhasedEscrow` the owner.

Reply:
Sound good, done.